internal/cmd/gocdk/testdata/deploy-help.ct

```diff
@@ -8,9 +8,10 @@ Usage:
   gocdk deploy <biome name> [flags]
 
 Flags:
-      --apply biome apply    whether to run biome apply before deploying (default true)
+      --apply                whether to run "biome apply" before deploying (default true)
   -h, --help                 help for deploy
       --image [name][:tag]   Docker image to deploy in the form [name][:tag]; name defaults to image name from Dockerfile, tag defaults to latest; empty string builds a new image
+      --verbose              true to print all output from Terraform during "biome apply"
```

Comment:
`-v` as well? It seems pretty common. Unless we are going to have other verbosity.

Reply:
Done, here and elsewhere.